gems/pending/appliance_console/logical_disk_volume.rb

```diff
@@ -0,0 +1,107 @@
+require 'active_record'
+require 'active_support/core_ext'
+require 'linux_admin'
+require 'pathname'
+require 'util/miq-password'
+require 'fileutils'
+require 'appliance_console/errors'
+require 'appliance_console/utilities'
+require 'appliance_console/logging'
+
+RAILS_ROOT ||= Pathname.new(__dir__).join("../../../")
+
+module ApplianceConsole
+  class LogicalDiskVolume
```

Comment:
I think `LogicalVolumeManagement` would be better. That is really what this does, and the backend it uses.